extensions/amp-experiment/0.1/variant.js

```diff
@@ -39,7 +39,7 @@ export function allocateVariant(win, config) {
 
   if (cidScope && config.consentNotificationId) {
     hasConsentPromise = userNotificationManagerFor(win)
-        .then(manager => manager.get(config.consentNotificationId))
+        .then(manager => manager.getNotification(config.consentNotificationId))
```

Comment:
Re: standup today. We may want to only wait for doc ready, not viewer is visible. Waiting for visible would defeat the point of render blocking.